interchaintest/helpers/query_helpers.go

```diff
@@ -22,3 +25,65 @@ func GetUnityContractWithdrawalReadyTime(t *testing.T, ctx context.Context, chai
 	require.NoError(t, err)
 	return res
 }
+
+// From stakingtypes.Validator
+type Vals struct {
```

Comment:
This is required here instead of `stakingtypes.Validator` due to the status failing (Read as a string enum const, but maps to a int32)